woocommerce/woocommerce-android
# Shipping Labels: print multiple packages

Part of #4444, adds support for printing multiple purchased packages.

<img width="360" alt="Screen Shot 2021-07-26 at 15 01 52" src="https://user-images.githubusercontent.com/1657201/127001763-7e3ca0cf-0611-45dc-adbd-126dbc78a9c0.png">

This depends on the FluxC PR: https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2063.

#### Testing
1. Create a multi product order eligible for shipping label creation.
2. Open the order in the app.
3. Click on Create shipping labels.
4. In the packaging step, make sure to move some items to different pacakges.
5. Purchase the shipping label.
6. Confirm that you see the correct UI for multiple packages in the print screen.
7. Click on "Print shipping labels"
8. Confirm that the downloaded PDF file contains all the labels.

Update release notes:
